e2e/paths/caseSubmitted_test.js

```diff
@@ -21,7 +20,7 @@ Scenario('Can give consent and submit the case', (I, caseViewPage, submitApplica
 });
 
 Scenario('Cannot submit a case unless consent is given', I => {
-  I.see(`I, ${config.swanseaLocalAuthorityUserOne.forename} ${config.swanseaLocalAuthorityUserOne.surname}, believe that the facts stated in this application are true.`);  
```

Comment:
@Jamesnelson117 can you look into configuring eslint to reject any trailing whitespaces? I am just annoyed to see this in code review from time to time.